doc/emergency-cpr.rst

```diff
@@ -0,0 +1,81 @@
+#############
+Emergency CPR
+#############
+
+Is Spyder not launching at all?
+The steps on this section should hopefully get it back up and running in no time.
+
+
+
+================
+Common solutions
+================
+
+#. Try :doc:`basic-first-aid` first, which usually resolves most Spyder install-related issues.
+
+#. **Make sure Spyder isn't already running** and no Spyder related windows (*e.g.* :doc:variableexplorer dialogs) are left open, and check that the preference setting :menuselection:`General --> Advanced Settings --> Use a single instance` isn't checked.
+
+   .. image:: images/emergency-cpr/emergency-cpr-single-instance.png
+      :alt: Spyder showing Use a single instance setting
+
+#. Try **starting Spyder via a different means**, such as from a shortcut, Anaconda navigator, or your command line (or Anaconda Prompt on Windows) by simply typing ``spyder`` then :kbd:`Enter`/:kbd:`Return`, and see if any of those work.
+   If so, then something's wrong with your install, not Spyder itself, and so we recommend following :ref:`troubleshooting-reinstalling-spyder-ref` to uninstall and reinstall Anaconda.
```

Comment:
No, that's intentional. If some start methods work but others don't, then the problem is almost certainly with the install/Anaconda rather than with Spyder itself.

Reply:
Ok, no problem. I misunderstood this one.